src/storage/txn/commands/rollback.rs

```diff
@@ -60,3 +78,94 @@ impl<S: Snapshot, L: LockManager> WriteCommand<S, L> for Rollback {
         })
     }
 }
+
+// #[cfg(test)]
```

Comment:
I cannot keep `#[cfg(test)]` here because in that case the compiler will complain:

> failed to resolve: could not find `tests` in `rollback`

Reply:
Yes. If it's used in other crates, even if only in tests, it cannot be marked `#[cfg(test)]`. You can remove it.